site/content/integrate/apps/deploy/deploy-aws/_index.md

```diff
@@ -0,0 +1,160 @@
+
+#### Set AWS_PROVISION_ environment variables
+
+Open a terminal where you installed the Apps plugin and set the following variables to the AWS credentials just created and saved:
+
+- `MM_APPS_PROVISION_AWS_ACCESS_KEY`  
+- `MM_APPS_PROVISION_AWS_SECRET_KEY`
+
+Set the following environment variables based on bucket name and region
+
+- `MM_APPS_S3_BUCKET`  
+- `MM_APPS_AWS_REGION`
+
+#### Initialize the AWS resources
+
+The following command will create Mattermost invoke credentials and policy for use with AWS.
+
+`go run ./cmd/appsctl aws init [--create] [--create-access-key] [...other flags]`
+
+It will create or update:
+
+- An IAM user that will be used by the server to invoke the app
+  (`mattermost-apps-invoke`).
+- An IAM group that the user belongs to, that will be used by the server to
+  invoke the app (`mattermost-apps-invoke-group`).
+- An IAM policy used to control access to the Lambda and S3 resources, used by
+  the invoke group (`mattermost-apps-invoke-policy`).
+- An IAM role used to execute app's functions
+  (`mattermost-apps-execute-lambda-role`) with the `AWSLambdaBasicExecutionRole`
+  attached to it.
+
+Flags:
+- `--create-access-key` create a new access key and print it out.
+- `--create` create resources that don't yet exist (user, group, role, access
+  policy).
+- `--group` name the group to invoke function(s), default
+  `mattermost-apps-invoke-group`.
+- `--policy` name the policy used to invoke function(s), default
+  `mattermost-apps-invoke-policy`.
+- `--user` name the user to invoke the function(s), default
+  `mattermost-apps-invoke`.
+
+The output of the command will contain two "Invoke" environment variables.
+
+### 2. Enable AWS Upstream for Mattermost Apps
+
+To enable AWS Apps installation and use, the Mattermost server must be
+(re-)started with the following environment variables:
+
+- `MM_APPS_AWS_ACCESS_KEY` An AWS Access Key ID for `mattermost-apps-invoke`
+  user. If `appsctl aws init --create-access-key` was used, it can be found
+  (once!) in the command's output.
+- `MM_APPS_AWS_SECRET_KEY` The Secret Key for the access key above.
+- `MM_APPS_AWS_REGION` The AWS region to use.
+- `MM_APPS_S3_BUCKET` The name of the AWS bucet for static assets and manifests.
+
+Restart the Mattermost server to complete your Mattermost and AWS setup. You can
+now deploy an app to AWS.
+
+
+### 3. Deploy apps to AWS
+
+#### Deploying an app to AWS
+
+After setting up your Mattermost instance, AWS key, and S3 bucket you can now
+deploy apps using `appsctl`. To install appsctl, `go install
+github.com/mattermost/mattermost-plugin-apps/cmd/appsctl@latest`
```

Comment:
might want to move the `appsctl` related install at the top since it applies not only to AWS.

Reply:
I'd like to further improve it by adding the `http` and `plugin` support to it, and then restructure the code a little, and the output with it. Another PR. Not 0.8.0

Reply:
I was just pointing out that the "To install appsctl..." section should prob. be moved out of here and placed at the top of the chapter?("# Deployment..") since it applies to more envs than AWS.